dynamo/plot/networks.py

```diff
@@ -357,6 +361,57 @@ def arcPlot(
         return ap
 
 
+def circos_plot(
+    network: nx.Graph,
+    node_label_key: str = None,
+    circos_label_layout: str = "rotate",
+    node_color_key: str = None,
+    show_colorbar=True,
+    edge_lw_scale: float = 0.5,
+    edge_alpha_scale: float = 0.5,
+) -> Axes:
+    """wrapper for drawing circos plot via nxviz >= 0.7.3
+
+    Parameters
+    ----------
+    network : nx.Graph
```

Comment:
Default numpy style (which dynamo uses) docstring is without indentation. Let me know if we should have indentation here.